### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by jrbourbeau**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,8 @@ dask>=2021.05.0
 distributed
 snowflake-connector-python[pandas]>=2.6.0
 snowflake-sqlalchemy
+# `pandas=2.2` dropped support for `sqlalchemy<2`, but `snowflake-sqlalchemy`
+# doesn't support `sqlalchemy<2` yet. Temporarily pinning `pandas<2.2` for now.
+# xref https://github.com/pandas-dev/pandas/issues/57049
+# xref https://github.com/snowflakedb/snowflake-sqlalchemy/issues/380
+pandas<2.2
```
Comment (jrbourbeau) [inline requirements.txt:9]: cc @phofl for visibility and in case you're aware of a better alternative

File: ci/environment-3.10.yaml [hunk 2]
```diff
@@ -6,8 +6,12 @@ dependencies:
   - python=3.10
   - dask
   - distributed
-  - pandas
-  - pyarrow<11
+  # `pandas=2.2` dropped support for `sqlalchemy<2`, but `snowflake-sqlalchemy`
+  # doesn't support `sqlalchemy<2` yet. Temporarily pinning `pandas<2.2` for now.
```
Comment (phofl) [inline ci/environment-3.10.yaml:10]:
```suggestion
  # doesn't support `sqlalchemy>2` yet. Temporarily pinning `pandas<2.2` for now.
```
?

File: requirements.txt (hunk 1 above)
Comment (phofl) [reply requirements.txt:9]: this is annoying..., thx

File: ci/environment-3.10.yaml (hunk 2 above)
Comment (jrbourbeau) [reply ci/environment-3.10.yaml:10]: Thanks -- typo

**Review by phofl — APPROVED** [review]

**Pull request merged**